cfme/scripting/tests/test_quickstart.py

```diff
@@ -62,6 +61,6 @@ def test_quickstart_run(image, python, root_volume, yamls_volume, check_docker):
     ({'a': 1}, {'a': 1}, []),
 ])
 def test_quickstart_version_changed(old, new, expected_changes):
-
+    from cfme.scripting import quickstart
```

Comment:
**REQUIRED:** We should add a comment explaining why we are loading this library inside a function rather than in module space.   Otherwise someone is going to unwittingly "fix" this and bring back the problem.